wandb/sdk/wandb_init.py

```diff
@@ -1123,6 +1124,10 @@ def init(
             for saving hyperparameters to compare across runs. The ID cannot
             contain the following special characters: `/\#?%:`.
             See [our guide to resuming runs](https://docs.wandb.com/guides/runs/resuming).
+        fork_from: (str, optional) A string with the format <run_id>?_step=<step> describing
+            a moment in a previous run to fork a new run from. Creates a new run that picks up
```

Comment:
```suggestion
            a point in a previous run to fork a new run from. Creates a new run that picks up
```
what do you think of using point instead of moment?